server/controllers/auth.controller.js

```diff
@@ -42,6 +43,12 @@ export function Login(req, res) {
         msg: 'Authentication failed. Passwords did not match.',
       });
     });
+
+    // for default error return
+    return res.json({
```

Comment:
due to the async nature of JavaScript.. this default return will be called and exeuted before the user.authentication's callback finishes. thus will introduce bug.

I believe if you try to login now, the code will break.

everytime we make any significant changes, do a little test of the module to make sure nothing breaks~

Thanks,
Kevin.
:)